src/getCoinCombination.test.js

```diff
@@ -7,4 +7,40 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return coorect combination for 2 cents`, () => {
```

Comment:
There is a typo in the test description. It should be 'correct' instead of 'coorect'.